pkgs/development/mobile/adb-sync/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, python, androidsdk, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "adb-sync-${version}";
+  version = "2016-08-31";
+
+  src = fetchgit {
+    url = "https://github.com/google/adb-sync";
+    rev = "7fc48ad1e15129ebe34e9f89b04bfbb68ced144d";
+    sha256 = "1y016bjky5sn58v91jyqfz7vw8qfqnfhb9s9jd32k8y29hy5vy4d";
+  };
+
+  preferLocalBuild = true;
+
+  buildInputs = [ python androidsdk makeWrapper ];
+
+  phases = "installPhase";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src/adb-channel $src/adb-sync $out/bin/
+
+    sed -i '1s|/usr/bin/python|${python}/bin/python|' $out/bin/adb-sync
```

Comment:
guess I didn't notice this before, but please use `${python.interpreter}` instead of `${python}/bin/python`. Also, I would expect the shebang already to be rewritten by `patchShebangs`.